src/fundus/publishers/de/frankfurter_rundschau.py

```diff
@@ -13,9 +14,15 @@
 
 class FrankfurterRundschauParser(ParserProxy):
     class V1(BaseParser):
-        _paragraph_selector = CSSSelector("p.id-StoryElement-paragraph, article > ul > li")
-        _summary_selector = CSSSelector("p.id-StoryElement-leadText")
-        _subheadline_selector = CSSSelector("h2.id-StoryElement-crosshead")
+        _paragraph_selector = XPath(
+            "//p[@class='id-StoryElement-paragraph'] | "
+            "//p[contains(@class,'id-Article-content-item-paragraph') and text()] |"
+            "//div[@class='id-Article-body']//ul/li[not(@class='id-AuthorList-item ')]"
```

Comment:
This one differs from the original `article > ul > li` is there a reason for that?

Reply:
Yes, they added another list item in some articles called "Meine News". You can see an example here: https://www.fr.de/politik/ampel-regierung-koalition-news-lindner-fdp-habeck-gruene-spd-wirtschaft-plan-papier-zr-93394586.html